### DISCUSSION TITLE
Icon Improvement for Unstable Rate Graph

### DISCUSSION THREAD

**Discussion by Theedoo** [ppy/osu#22064] [Ideas] [answered]
Instead of the current "Bus & Bike" model;
![image](https://user-images.githubusercontent.com/65991538/211169862-7e254374-1f08-43b3-b102-9bb11f8241b6.png)
Why not have a "Cheetah & Snail" model (that I believe was previously implemented, but phased out?) as it both is: more visually appealing (having two goofy little animals,) and is more immediately understood (instead of a faster vehicle vs. a decently quick vehicle)
Ex.
![image](https://user-images.githubusercontent.com/65991538/211171636-1895672c-2b9b-4f9b-9a2c-913121184301.png)
With the cheetah facing left, meaning left is faster (likely including "speed lines",) and the snail facing right (meaning right is slower, or a flipped alternative)
I do believe the cheetah is not a "globally" known "fastest (land) animal," so I feel it'd be acceptable changing it out to a horse, or a car. Even if it isn't, I do believe most regions in the world do know the snail as a "very slow (land) animal," so in comparison they should be able to deduce a cheetah is faster.

This whole argument is more of a visual/mental issue, with the "Bus & Bike" model being both unappealing and slightly confusing, whereas the "Cheetah & Snail" model being more visually appealing, and more easily understood.

**Comment by frenzibyte** [comment, marked as answer]
There is an existing issue thread for this (https://github.com/ppy/osu/issues/11915). Best to put your proposal there to keep things in one place.
